core/tests/scheduler_cost_adjustment.rs

```diff
@@ -124,7 +124,9 @@ impl TestSetup {
             },
             Err(err) => {
                 unreachable!(
-                    "All test Transactions should be well-formatted for execution and commit, err: '{}'", err
+                    "All test Transactions should be well-formatted for execution and commit, \
+                     err: '{}'",
+                    err
```

Comment:
I'm going to manually adjust this one, not sure why the linter didn't complain about it. Maybe the **`** (single quotes) threw it off ?